### ISSUE TITLE
Europa FM Spain stream not detected

### ISSUE THREAD

**Issue by erduende** [segler-alex/radiobrowser#87]
Hello, I just changed Europa FM Spain stream URL (http://icecast-streaming.nice264.com/europafm) an it is not recognized by radiobrowser but it works if you use it directly in a web browser.

**Comment by segler-alex** [comment]
i will check when i come home

**Comment by erduende** [comment]
Thx! The station ID is 87897

**Comment by segler-alex** [comment]
if i click the link in my browser, he asks me for login information, http 401.
hmmm. does it still work for you?
maybe it is geoblocked

**Comment by segler-alex** [comment]
when i try to go to http://www.europafm.com/directo/ it still asks me for authentication :(
my best bet is, that it only works in your country.

**Comment by erduende** [comment]
Right, it must be geolocation restriction because I can listen it in Spain. In this case, how can I add the station stream URL to the database?

**Comment by JOM13** [comment]
Via TuneIn I could find a stream which works outside Spain (Colombia, in my case): http://212.129.60.86:9968/;
